doc/source/whatsnew/v1.1.0.rst

```diff
@@ -36,6 +36,31 @@ For example:
    ser["2014"]
    ser.loc["May 2015"]
 
+
+.. _whatsnew_110.groupby_key:
+
+Allow NA in groupby key
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+We've added a ``dropna`` keyword to :meth:`DataFrame.groupby` and :meth:`Series.groupby` in order to
+allow ``NA`` values in group keys. Users can define ``dropna`` to ``False`` if they want to include
+``NA`` values in groupby keys. The default is set to ``True`` for ``dropna`` to keep backwards
+compatibility (:issue:`3729`)
```

Comment:
add a ref to the new doc-section.

Reply:
emm, what does this mean?

Reply:
you add something like: ```:ref:`cookbook<cookbook.grouping>` ``` to put in a link to the new groupby section you added about (obviously use the new ref you created)

Reply:
thanks for the hint! I added one following other examples, pls let me know if it is okay now